test/common/tests.go

```diff
@@ -71,6 +71,9 @@ var (
 				/*FLAKY on RHMI*/ {"Verify Alerts are not firing during or after installation apart from DeadMansSwitch", TestIntegreatlyAlertsFiring},
 				{"Verify prometheus metrics scrapped", TestMetricsScrappedByPrometheus},
 				{"A27 + A28 - Verify pod priority class is created and set", TestPriorityClass},
+				{"H08 - Verify that backend can be created via backend CR", Test3scaleBackendViaCR},
+				{"H09 - Verify that product can be created via tenant CR", Test3scaleProductViaCR},
+				{"H10 - Verify that tenant can be created via tenant CR", Test3scaleTenantViaCr},
```

Comment:
We already [have test cases with these IDs](https://github.com/integr8ly/integreatly-operator/tree/master/test-cases/tests/products)

```suggestion
				{"H29 - Verify that backend can be created via backend CR", Test3scaleBackendViaCR},
				{"H30 - Verify that product can be created via tenant CR", Test3scaleProductViaCR},
				{"H31 - Verify that tenant can be created via tenant CR", Test3scaleTenantViaCr},
```

could we also add test cases for these automated tests [here](https://github.com/integr8ly/integreatly-operator/tree/master/test-cases/tests/products) and mark them as automated? Thanks!

(A guide for adding new test cases is [here](https://github.com/integr8ly/integreatly-operator/tree/master/test-cases#how-to-create-a-test-case))